style.css

```diff
@@ -92,3 +95,205 @@ header {
   color: #505f79;
   font-size: 20px;
 }
+
+/* -----work section----- */
+
+.works {
+  margin: 40px 0;
+  padding: 20px;
+  display: grid;
+  grid-template-columns: auto;
+  gap: 20px;
+}
+
+.project-card {
+  border: 1px solid #ebebff;
+  border-radius: 10px;
+  width: 95%;
+  margin: 8px auto;
+  background-color: #fff;
+  padding: 10px;
+}
+
+.canopy {
+  color: #344563;
+  margin-left: 12px;
+}
+
+.project-img {
+  display: flex;
+  justify-content: center;
+}
+
+.project-card img {
+  margin: 10px 0;
+}
+
+.project-title {
+  display: flex;
+  justify-content: start;
+  margin: 10px;
+}
+
+.project-title h2 {
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+/* .project-card span, img {
+  margin-right: 10px;
+  display: flex;
+  line-height: 16px;
+  font-size: 13px;
+} */
+
+/* .info {
+  display: flex;
+  align-items: center;
+  /* margin: 0 5px; */
+
+/* justify-content: center; */
+
+/* border: 1px solid red;
+  padding-right: 10px;
+} */
+
+.info {
+  margin: 0 auto;
+
+  /* display: flex; */
+
+  /* align-items: center;  */
+
+  /* justify-content: center; */
+}
+
+.info img,
+span {
+  margin-bottom: 0;
+  margin-right: 5px;
+  color: #7a869a;
+  font-size: 13px;
+}
+
+/* .info li {
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  margin: 0 10px;
+} */
+
+.info .color {
+  color: #344563;
+}
+
+/* .faint {
+  color: #7a869a;
+  border: 1px solid red;
+} */
+
+/* span { */
+
+/* background-color: #7a869a; */
+
+/* width: 8px; */
+
+/* height: 8px; */
+
+/* border-radius: 50 */
+
+.project-paragraph {
+  display: flex;
+  text-align: start;
+  margin: 10px;
+  width: 292px;
+}
+
+.project-paragraph p {
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.technologies {
+  display: flex;
+  margin: 10px 10px;
+}
+
+.technologies li {
+  margin-right: 10px;
+  padding: 5px 8px;
+  background-color: #ebebff;
+  border-radius: 5px;
+  color: #6070ff;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+}
+
+.project-btn {
+  margin: 10px 10px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  color: #396df2;
+  border: 1px solid #396df2;
+  border-radius: 8px;
+  background-color: #fff;
+  padding: 10px;
+}
```

Comment:
- [ ] In the original design, the button is aligned to the left with the icons. You could consider using `margin: 10px 0` instead.
- [ ] All buttons should have their corresponding action stylings, such as active, hover, and pressed. You can check out how they should look in the Figma design:

![image](https://user-images.githubusercontent.com/66967274/143680615-9a97abf7-b32f-49fe-9234-7ae7d73797cd.png)

- [ ] Those actions should be added to all buttons.